Monika After Story/game/script-topics.rpy

```diff
@@ -15751,3 +15751,44 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_knowitsyou",category=['you'],prompt="How do you know it's always me talking to you?",pool=True))
+
+label monika_knowitsyou:
+if mas_isMoniEnamored(higher=True):
+    m 1euc "Well, if I'm being entirely honest..."
+    m 1eud "I don't really have a way of knowing."
+
+        if seen_event("monika_MC_is_gone"):
+            m 3lka "I already told you that I can't really see into your world."
+
+    m 3eka "I can tell when you're in the room with me, but that's all I know."
+    m "For all I know, it could be anyone looking at me right now. You could have other people with you."
+    m 2dua "...But I trust you, [player]."
+    m "I trust that you aren't leaving me alone because {i}you{/i} care about me."
+    m 2ekb "And if someone else {i}is{/i} looking in right now, I choose to trust that it's because you've given them a good reason to."
+    m 2eka "Maybe someday I'll be able to know for sure, but until then that's enough for me."
+    
+elif mas_isMoniHappy(higher=True):
+    m 1rksdlb "Ehehe...That's an odd question to ask, [player]."
```

Comment:
```suggestion
    m 1rksdlb "Ehehe... That's an odd question to ask, [player]."
```
There should be a space after a sentence-ending ellipsis. I think.

Reply:
I'm going to go look through the script to make sure, but I thought MAS dialogue never puts a space, even though that'd be grammatically correct.

Thank you for pointing this out to me though, I definitely need to double-check. :D

Reply:
Rule of thumb:
If it's a new sentence after the ellipsis, space and the first word after the ellipsis will be capitalized.
If it's a continuation off the last one, then there is no space, and the word is lowercase.

If it extends over multiple lines, then it is always capitalized, unless the last line ends in an ellipsis, and the next one starts with `...and`

Reply:
Ah, thank you! Good to know, I'll keep that in mind for the future. Committing this fix now then~